first-task.html

```diff
@@ -9,6 +9,20 @@
 </head>
 
 <body>
+
+  <header class="header">
+    <div class="header-logo">
+      <img src="img/Ellipse 1.png">
+      <a class="header-text"> LOGO </a>
+    </div>
+    <div class="header-info">
+      <a class="header-home">Home</a>
+      <a class="header-blogs">Blogs</a>
+      <a class="header-about">About</a>
+      <a class="header-contact">Contact Us</a>
+    </div>
+      <button class="header-button"> Subscribe </button>
```

Comment:
```suggestion
    <div>
      <div class="header-info">
        <a class="header-home">Home</a>
        <a class="header-blogs">Blogs</a>
        <a class="header-about">About</a>
        <a class="header-contact">Contact Us</a>
      </div>
      <button class="header-button">Subscribe</button>
    </div>
```
Instead of that huge margin left, you could wrap these two blocks into a block. You can adjust space between the navigation bar and the button by setting margin-left for the button